apps/api/src/form/form.service.ts

```diff
@@ -26,4 +27,45 @@ export class FormService {
       confirmationId: id,
     });
   }
+
+  async getAllForms(): Promise<FormEntity[]> {
+    return await this.formRepository.find({});
+  }
+
+  async flattenAndTransformFormData(forms: FormEntity[]): Promise<FormExportColumns[]> {
+    return forms.map(form => {
+      return {
+        deployAnywhere: booleanToYesNo(form.payload.availabilityInformation.deployAnywhere),
```

Comment:
it might look better to destructure `form` to get `payload`.

Reply:
done